workflow

```diff
@@ -288,17 +288,22 @@ preferred.
 
 ### Install and Boot into the new system
 
-One of the easiest ways to test your changes is to boot a test system onto your
-new bits. No matter what kind of change you've made, this should work. There are
-various ways to do this based on your distribution; however, for most folks this
-can be achieved via the build tool onu(1ONBLD). If you're inside of bldenv and
-have finished your build, the simplest way to do this is:
+One of the easiest ways to test your changes is to boot a test system
+onto your new bits. No matter what kind of change you've made, this
+should work. There are various ways to do this based on your
+distribution; however, for most folks this can be achieved via the
+build tool onu(1ONBLD). If you've finished your build and are still in
+the shell started by `bldenv`, the simplest way to do this is:
 
 ```
-# onu -t new-nightly-be
+# onu -t new-nightly-be -d $PKGARCHIVE
 #
 ```
 
+where `$PKGARCHIVE` is set to `${CODEMGR_WS}/packages/${MACH}/nightly`
+(for a DEBUG build) or `${CODEMGR_WS}/packages/${MACH}/nightly-nd`
+(for a non-DEBUG build).
```

Comment:
Can we leave a note that this is set by `bldenv`? When I first read this I read it as someone was having to manually set this.

Reply:
Done.